### ISSUE TITLE
Multiple garments?

### ISSUE THREAD

**Issue by alelordelo** [MengZephyr/DynamicNeuralGarments#3]
Thanks for sharing this amazing research!

Have you tried an avatar with multiple garments (ex, pants, shirt and jacket). Would it work?

**Comment by MengZephyr** [comment]
Yes, it does work. Most garments in our paper have multi layers. Our method also works well in cases of pants, t-shirt and jacket.
<img width="274" alt="image" src="https://user-images.githubusercontent.com/28139225/187587814-e5e12181-4dd7-4e5f-9ebe-e7598653bbf7.png">

**Comment by alelordelo** [comment]
Thanks @MengZephyr.

I mean 2 layers on top, lets say a jacket on top of a shirt. I am wondering how the model would generalise an unseen combination...

Ex: you train on a jacket dataset. Then apply a shirt layering with the jacket, that would probably mess the jacket training model?
